packages/flutter_tools/lib/src/isolated/native_assets/native_assets.dart

```diff
@@ -410,417 +565,363 @@ Uri nativeAssetsBuildUri(Uri projectUri, OS os) {
   return projectUri.resolve('$buildDir/native_assets/$os/');
 }
 
-class HotRunnerNativeAssetsBuilderImpl implements HotRunnerNativeAssetsBuilder {
-  const HotRunnerNativeAssetsBuilderImpl();
+/// With `flutter run -d all` we need a place to store the native assets
```

Comment:
Nit: The diff is larger than it needs to be due to reorderings. This makes reviewing harder and redirects questions to you via git blame. If you feel strongly about the order of code, fine be me.